### DISCUSSION TITLE
Automatically import existing CVEs as GitHub advisories

### DISCUSSION THREAD

**Discussion by adulau** [community/community#14321] [Code Security]
Hi All,

We have various open source projects such as [MISP](https://github.com/misp) and we would like to import [our existing CVEs](https://www.misp-project.org/security/#advisories) as GitHub advisories. We were wondering if there is a way to do it and especially to do it automatically using all the fields from the already assigned CVEs.  I searched documentations and existing support/feedback without finding any reference for that. and if there is no automatic way to do it, is there an API to feed the GitHub security advisories?

Thank you very much.

**Comment by taladrane** [comment]
Hi @adulau, thank you for the feedback! I wanted to let you know that we have been aware of this gap in our database and working on fixing it. Just this week we finished this project and have [backfilled all historical CVEs that were missing in the Advisory Database](https://github.blog/changelog/2022-06-08-all-historical-nvd-advisories-are-now-listed-on-github/) in the unreviewed state 🎉 for more information on making suggestions to advisories in the database, please see [this docs article](https://docs.github.com/en/code-security/dependabot/dependabot-alerts/editing-security-advisories-in-the-github-advisory-database).

**Reply by adulau** [reply]
Thanks a lot, this is awesome! I maybe have a stupid question. How do we move the unreviewed CVE into a specific repo? I see the proper commit references which is great. I wanted to do it for MISP/MISP repository.
